libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.html

```diff
@@ -17,6 +17,7 @@
     (mouseover)="_setActive($event, false)"
     (mouseout)="_setActive($event, true)"
     [attr.aria-selected]="selectionMode === 'none' ? null : !!_selectedItem || _selectedItem === 0"
+    [attr.aria-describedby]="ariaDescribedby"
```

Comment:
What's the purpose of this? The aria-describedby attribute is used to indicate the IDs of the elements that describe the object. Will there be an element that will have an id that describes this element?

Reply:
Adding this attribute to address ISSUE 12 from the attached docs
> Ideally the sub header should be associated to the list and should be read on navigating to the grid list.

As the most straightforward way of solving this, I'm adding an option to explicitly tie the header with corresponding options. Let me explain why.

Current structure of elements within the grid is as follows

```
<fd-grid-list-group-header>Header 1</fd-grid-list-group-header>
<fd-grid-list-item>....</fd-grid-list-item>
<fd-grid-list-item>....</fd-grid-list-item>
<fd-grid-list-item>....</fd-grid-list-item>
<fd-grid-list-group-header>Header 2</fd-grid-list-group-header>
<fd-grid-list-item>....</fd-grid-list-item>
<fd-grid-list-item>....</fd-grid-list-item>
<fd-grid-list-item>....</fd-grid-list-item>
```

Because of this I don't see any reliable ways to have an implicit association of items with their headers. In my opinion, this can be properly solved in 2 ways:
- by adding some fuzzy js logic to determine which header a particular item belongs to
- by changing the `fd-grid-list-group-header` to `fd-grid-list-group`, that would contain all corresponding items instead of keeping it as a sibling of those items (this would be the best IMO)

Both these options require significant changes, which would be too much for the scope of particular PR. If you think that's reasonable, I can create a separate issue to address this point in the way you see fit. 

Please let me know

Reply:
I see. You can leave the current implementation

Reply:
Thanks for explaining